core/src/apps/webauthn/fido2.py

```diff
@@ -1739,16 +1712,10 @@ def cbor_get_assertion_sign(
     )
 
     # Sign the authenticator data and the client data hash.
-    dig = hashlib.sha256()
-    dig.update(authenticator_data)
-    dig.update(client_data_hash)
-    if user_presence:
-        privkey = cred.private_key()
-    else:
-        # Spec deviation: Use a bogus key during silent authentication.
-        privkey = _BOGUS_PRIV_KEY
```

Comment:
The declaration of the variable  `_BOGUS_PRIV_KEY` can be removed.

Reply:
Done.